src/Simulation/AutoSubstitution.Integration.Tests/Tests.Microsoft.Quantum.AutoSubstitution.Integration.csproj

```diff
@@ -29,7 +29,10 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <ProjectReference Include="..\CSharpGeneration\Microsoft.Quantum.CSharpGeneration.fsproj" PrivateAssets="All" IsQscReference="true" />
+    <PackageReference Include="Microsoft.Quantum.CSharpGeneration" Version="0.17.2106147878-beta" IsQscReference="true" />
```

Comment:
If we remove `<CSharpGeneration>false</CSharpGeneration>`, can we remove this `PackageReference`?

Reply:
Ah, that's a good idea! I hadn't noticed that. I'll do a pass through and fix up the ones you pointed out and see if there any others that could benefit from that.

Reply:
Darn... that doesn't work that well, because of the fact that CSharpGeneration will cause the include of the package Runtime.Core: 
https://github.com/microsoft/qsharp-compiler/blob/6bf4159e57b11e6046ebc68f297e939637d53641/src/QuantumSdk/Sdk/Sdk.props#L71-L72

Reply:
Maybe if we add a property flag that lets us suppress the inclusion of the Runtime.Core package? It would simplify some of this usage here...

Reply:
I'm attempting to add this functionality with https://github.com/microsoft/qsharp-compiler/commit/fc1d0ae0158c6eecdcb5daaed900c56b8cbd136f

Reply:
Ok, I was finally able to resolve this (and fix a bug in the process). The required package inclusions are now much simpler.